ibm/config.go

```diff
@@ -1384,7 +1384,14 @@ func (c *Config) ClientSession() (interface{}, error) {
 		Authenticator: authenticator,
 	}
 	vpcclient, err := vpc.NewVpcV1(vpcoptions)
-	if err != nil {
+	if err == nil {
+		// Enable retries for API calls
+		vpcclient.Service.EnableRetries(c.RetryCount, c.RetryDelay)
```

Comment:
this is called twice in the code 
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/1756c6e10323b31f698efd7d597b26821d37ab01/ibm/config.go#L1389
 https://github.com/IBM-Cloud/terraform-provider-ibm/blob/1756c6e10323b31f698efd7d597b26821d37ab01/ibm/config.go#L1398
 
 Make sure it is called at one place